.storybook/4.SHADOW.stories.mdx

```diff
@@ -217,7 +217,7 @@ Using both size and color tokens, different shadows can be applied to components
         boxShadow: 'var(--shadow-size-lg) var(--color-shadow-default)',
       }}
     >
-      <span>Modal</span>
+      Modal
     </div>
   </div>
   <div
```

Comment:
Updates to fix text in dark mode

<img width="1512" alt="Screenshot 2023-11-11 at 7 48 35 AM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/f8c2e881-3b1f-42a6-a373-f1d62ebcde8f">
<img width="1512" alt="Screenshot 2023-11-11 at 7 49 03 AM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/30d474f0-d4fb-421f-9830-94ced1339adc">